charts/seed-bootstrap/charts/etcd-druid/templates/druid-clusterrole.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: ClusterRole
+metadata:
+  name: gardener.cloud:system:etcd-druid
+  labels:
+{{ toYaml .Values.labels | indent 4 }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - configmaps
+  verbs:
+  - get
+  - list
+  - patch
+  - update
+  - watch
+  - create
```

Comment:
Won't it require `delete` permission?

Reply:
Yes. Added.

Reply:
`delete` verb is still missing for `services` and `configmaps`.

Reply:
Done